brunch-config.js

```diff
@@ -1,64 +1,64 @@
-var neat = require("bourbon-neat").includePaths
-var bourbon = require("bourbon").includePaths
+var neat = require('bourbon-neat').includePaths
```

Comment:
Missing semicolon.

[Read more about it here.](http://eslint.org/docs/rules/semi)